src/clj_kondo/impl/cache.clj

```diff
@@ -136,9 +136,9 @@
                 idacs
                 [:clj :cljs :cljc])]
     (reduce (fn [idacs lang]
-              (update-in idacs [lang :defs]
-                         (fn [defs]
-                           (update-defs idacs cache-dir lang defs)))
+              #_(update-in idacs [lang :defs]
```

Comment:
Unused, was hesitant about deleting it

Reply:
I would be pretty surprised if this code is really unused.

Reply:
Indeed, the lack of `!` (while there are other `!`s in code) confused me

Amending